src/code/z_actor.c

```diff
@@ -1514,14 +1495,15 @@ u32 func_8002F194(Actor* actor, GlobalContext* globalCtx) {
     return false;
 }
 
-s32 func_8002F1C4(Actor* actor, GlobalContext* globalCtx, f32 arg2, f32 arg3, u32 exchangeItemId) {
+s32 Actor_RequestToTalkAndExchangeItemInRange(Actor* actor, GlobalContext* globalCtx, f32 radius, f32 height,
```

Comment:
Yeah the reason why it was named this way is because there were similar functions and I wanted to highlight the difference between them. Maybe `Actor_RequestToTalkTradeItemInRange` instead? Still a bit long unfortunately.